src/js/node/http.ts

```diff
@@ -138,7 +145,8 @@ var FakeSocket = class Socket extends Duplex {
   address() {
     // Call server.requestIP() without doing any propety getter twice.
     var internalData;
-    return (this.#address ??= (internalData = this[kInternalSocketData])?.[0]?.requestIP(internalData[2]) ?? {});
+    return (this.#address ??=
+      (internalData = this[kInternalSocketData])?.[0]?.[serverSymbol].requestIP(internalData[2]) ?? {});
```

Comment:
Why is this an array now?

Reply:
its not an array, `kInternalSocketData` was changed from holding a `bun.Server` to an `http.Server` so there's a `serverSymbol` access